plugin/resolver/timeout/wrappers!.gotpl

```diff
@@ -2,32 +2,34 @@
 {{ reserveImport "time"  }}
```

Comment:
file name became `wrappers!.gotpl`

Reply:
it's fine !.gotpl templates goes first in the graphql template rending engine

``` go
		// important files go first
		if strings.HasSuffix(roots[i], "!.gotpl") {
			return true
		}

```

Reply:
up